src/amo/components/SearchSort/SearchSort.scss

```diff
@@ -0,0 +1,82 @@
+@import '~core/css/inc/mixins';
+@import '~core/css/inc/vars';
+
+@mixin arrow-link-right-aligned() {
+  background-image: url('~amo/img/icons/arrow-link-color.svg');
+  margin-right: 0;
+  position: absolute;
+  right: 8px;
+  top: 14px;
+}
+
+.SearchSort {
+  background: $base-color;
+  border-radius: $border-radius-default;
+  border: 0;
+  margin: 0 5% 10px;
+  overflow: hidden;
+}
+
+.SearchSort-toggle {
```

Comment:
These styles aren't specific to a `SearchSort`, just that dropdown component. It would be nice to move this into a `Picker` or `Dropdown` element or something (maybe prefix with `Inline` but that could be screen size dependent).